jmx/tests/org.fusesource.ide.jmx.camel.tests.integration/src/org/fusesource/ide/jmx/camel/tests/integration/navigator/RemoteDebugWhenEditingRoutesFromJMXNavigatorIT.java

```diff
@@ -57,11 +57,14 @@ public void tearDown() throws Exception {
 			ExtensionManager.getProvider(DefaultConnectionProvider.PROVIDER_ID).removeConnection(jmxConnection);
 		}
 		projectWithDebugAvailableDeployedHelper.clean();
+		if (remoteDebuglaunch != null) {
+			remoteDebuglaunch.terminate();
```

Comment:
what do you think to also try to kill processes retrieved by remoteDebuglaunch.getProcesses() ?